source/ed/osm2ed.cpp

```diff
@@ -298,12 +298,14 @@ void OSMCache::insert_ways(){
         values.push_back(std::to_string(way.osm_id));
         values.push_back(way.name);
         values.push_back("way:"+std::to_string(way.osm_id));
+        values.push_back("");
+        values.push_back("true");
```

Comment:
Should you not take the value from the way instead ? 

```suggestion
        values.push_back(way.visible);
```

Reply:
OSM doesn't have that kind of information, I guess.

Reply:
OSM and Geopal use the same bdd struct _georef.way_ to keep `ways`. 
to work, geopal needs visible field.
so, OSM has to handle it too. but OSM hasn't information about visibility. That why
`values.push_back("true");` all the time